PSReadLine/Prediction.Views.cs

```diff
@@ -273,6 +303,36 @@ internal PredictionListView(PSConsoleReadLine singleton)
             internal override bool HasPendingUpdate => _updatePending;
             internal override bool HasActiveSuggestion => _listItems != null;
 
+            /// <summary>
+            /// Calculate the max width and height of the list view based on the current terminal size.
+            /// </summary>
+            private (int maxWidth, int maxHeight, bool extraCheck) RefreshMaxViewSize()
```

Comment:
`extraCheck` here is too ambiguous.  Looking at the usage below, would it be better to call it `heightCheck`?

Reply:
Changed the name to be `checkOnHeight`.